packages/map/src/CdcMap.tsx

```diff
@@ -394,9 +395,13 @@ const CdcMap = ({
 
       // Special Classes (No Data)
       if (result[legendIdx].special) {
-        const specialClassColors = chroma.scale(['#D4D4D4', '#939393']).colors(specialClasses)
-
-        return specialClassColors[legendIdx]
+        if (versionNeedsUpdate(state.version, '4.24.11')) {
```

Comment:
@adamdoe this is a big nit, but it's kinda confusing that this says versionNeedsUpdate. like that to me implies that you're going to do something to update it. could we possible alias that function as something more obvious? like maybe `needsLegacySupport(state.version, '4.24.11')`

Reply:
like we could export another function from the versionNeedsUpdate function that is just this alias, that way others will know the other name.

we could pick a more universal name too like `isOlderVersion`

We could do this in a totally different PR too, just wanted to point it out here.